bld/namelist_files/namelist_definition.xml

```diff
@@ -220,32 +220,30 @@
        Default: FALSE
        </entry>
 
-<entry id="Nudge_Path" type="char*256" input_pathname="abs" category="nudging"
+<entry id="Nudge_Filenames" type="char*256(1000)" category="nudging"
        group="nudging_nl" valid_values="" >
-       Full pathname of analyses data to use for nudging.
-        (e.g. '/$DIN_LOC_ROOT/atm/cam/nudging/')
+       Full pathnames of analyses data to use for nudging.
```

Comment:
I think this should be:
```suggestion
       List of the absolute paths of the filenames of analyses data to use for nudging.
```

Reply:
I have backed out the nudging.F90 code and associated namelist - so I will take care of this in the nudging PR.